coopterrones/color-palette
# Test/integration tests

#### What's this PR do?
- adds integration testing in `App.test.js`

#### Where should the reviewer start?
- Reviewer should start in `App.test.js`

#### How should this be manually tested?
- check for all tests passing 
- check coverage of tests
